optimize/backend/src/test/java/io/camunda/optimize/service/util/BpmnModelUtilTest.java

```diff
@@ -0,0 +1,106 @@
+  public static final String SERVICE_TASK_TYPE = "serviceTask";
+
+  public static final String SIMPLE_SERVICE_TASK_PROCESS =
+      Bpmn.convertToString(createSimpleServiceTaskProcess(CUSTOMER_ONBOARDING));
+
+  @Test
+  void shouldParseBpmnModel() {
+    final BpmnModelInstance modelInstance =
+        BpmnModelUtil.parseBpmnModel(SIMPLE_SERVICE_TASK_PROCESS);
+    assertThat(modelInstance).isNotNull();
+  }
+
+  @Test
+  void shouldExtractFlowNodeData() {
+    final List<FlowNodeDataDto> flowNodeData =
+        BpmnModelUtil.extractFlowNodeData(SIMPLE_SERVICE_TASK_PROCESS);
+    assertThat(flowNodeData).isNotNull();
+    assertThat(flowNodeData.isEmpty()).isFalse();
+    FlowNodeDataDto endEventflowNodeDataDto = flowNodeData.get(0);
+    assertThat(endEventflowNodeDataDto.getId()).isNotNull();
+    assertThat(endEventflowNodeDataDto.getId()).isEqualTo(END_EVENT);
+    assertThat(endEventflowNodeDataDto.getType()).isEqualTo(END_EVENT_TYPE);
+
+    FlowNodeDataDto serviceTaskflowNodeDataDto = flowNodeData.get(1);
+    assertThat(serviceTaskflowNodeDataDto.getId()).isNotNull();
+    assertThat(serviceTaskflowNodeDataDto.getId()).isEqualTo(SERVICE_TASK);
+    assertThat(serviceTaskflowNodeDataDto.getName()).isEqualTo(SERVICE_TASK);
+    assertThat(serviceTaskflowNodeDataDto.getType()).isEqualTo(SERVICE_TASK_TYPE);
+
+    FlowNodeDataDto startEventflowNodeDataDto = flowNodeData.get(2);
+    assertThat(startEventflowNodeDataDto.getId()).isNotNull();
+    assertThat(startEventflowNodeDataDto.getId()).isEqualTo(START_EVENT);
+    assertThat(startEventflowNodeDataDto.getName()).isEqualTo(START_EVENT);
+    assertThat(startEventflowNodeDataDto.getType()).isEqualTo(START_EVENT);
+  }
+
+  @Test
+  void shouldExtractUserTaskNames() {
+    final String bpmnModelInstance =
+        Bpmn.convertToString(createSimpleUserTaskProcess(CUSTOMER_ONBOARDING));
+    final Map<String, String> userTaskNames = BpmnModelUtil.extractUserTaskNames(bpmnModelInstance);
+    assertThat(userTaskNames).isNotNull();
+    assertThat(userTaskNames.isEmpty()).isFalse();
+
+    assertThat(userTaskNames.containsKey(USER_TASK)).isTrue();
+  }
+
+  @Test
+  void shouldExtractProcessDefinitionName() {
+    final Optional<String> processName =
+        BpmnModelUtil.extractProcessDefinitionName(
+            CUSTOMER_ONBOARDING, SIMPLE_SERVICE_TASK_PROCESS);
+    assertThat(processName.isPresent()).isTrue();
+    assertThat(CUSTOMER_ONBOARDING).isEqualTo(processName.get());
+  }
+
+  @Test
+  void shouldExtractFlowNodeNames() {
+    final List<FlowNodeDataDto> flowNodeData =
+        BpmnModelUtil.extractFlowNodeData(SIMPLE_SERVICE_TASK_PROCESS);
+    final Map<String, String> flowNodeNames = BpmnModelUtil.extractFlowNodeNames(flowNodeData);
+    assertThat(flowNodeNames).isNotNull();
+    assertThat(flowNodeNames.isEmpty()).isFalse();
+
+    assertThat(flowNodeNames.containsKey(START_EVENT)).isTrue();
+    assertThat(START_EVENT).isEqualTo(flowNodeNames.get(START_EVENT));
```

Comment:
```suggestion
   // when
    final List<FlowNodeDataDto> flowNodeData =
        BpmnModelUtil.extractFlowNodeData(SIMPLE_SERVICE_TASK_PROCESS);
    
    // then
    final Map<String, String> flowNodeNames = BpmnModelUtil.extractFlowNodeNames(flowNodeData);
    assertThat(flowNodeNames).isNotNull();
    assertThat(flowNodeNames.isEmpty()).isFalse();

    assertThat(flowNodeNames.containsKey(START_EVENT)).isTrue();
    assertThat(START_EVENT).isEqualTo(flowNodeNames.get(START_EVENT));
```

Another example of how we comment. Please apply it to all tests

Reply:
fixed